x-pack/plugins/lens/public/shared_components/legend_settings_popover.test.tsx

```diff
@@ -38,6 +38,7 @@ describe('Legend Settings', () => {
       mode: 'auto',
       onDisplayChange: jest.fn(),
       onPositionChange: jest.fn(),
+      onLegendSizeChange: jest.fn,
```

Comment:
Why above we use `jest.fn()` but here just `jest.fn`?

Reply:
It's a typo, thanks for noticing!